### DISCUSSION TITLE
vercel open command

### DISCUSSION THREAD

**Discussion by dferber90** [vercel/vercel#8174] [Ideas] [answered]
While developing in a project, I often find myself wanting to open that project on `vercel.com`, for example to edit env vars or to see the latest deployment.

At the moment my solution is to open `vercel.com/:team/:project` manually, which is a slight annoyance.

I wish there was a `vc open` command which would open the linked project in a new browser tab.

```bash
vc open # opens vercel.com/:teamSlug/:projectSlug
```

It could optionally allow opening sections via subcommands

```bash
vc open settings # opens vercel.com/:teamSlug/:projectSlug/settings
vc open analytics # opens vercel.com/:teamSlug/:projectSlug/analytics
vc open deployments # opens vercel.com/:teamSlug/:projectSlug/deployments
```

*Update: Just found out about https://github.com/vercel/vercel/pull/8072*

**Comment by styfle** [comment, marked as answer]
I think `vc inspect` should be updated to handle this use case. It could be changed so its scoped to the current project like many other commands.
